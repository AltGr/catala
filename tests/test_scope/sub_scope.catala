/*
declaration scope O:
  context o content bool

declaration scope A:
  context a content int
  context b content bool
  context a_base content int
  context scopeO scope O

declaration scope B:
  context a content int
  context b content bool
  context scopeA scope A
  context scopeAbis scope A

scope A:
  def a_base = 1
  def a = -1
  def b = a > 0

scope B:
  def a = 42
  def b = scopeA.b
  def scopeA.a ? a > 0 ~= scopeA.a_base 
*/
