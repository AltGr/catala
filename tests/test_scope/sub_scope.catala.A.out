[RESULT] a -> -1
[RESULT] b -> false
[RESULT] a_base -> 1
